README.md

```diff
@@ -65,6 +65,56 @@ use {
 
 If you use another plugin manager just make sure `tabout.nvim` is loaded after `nvim-treesitter` and any completion that already uses your _tabkey_.
 
+### [lazyvim](https://www.lazyvim.org/)
```

Comment:
@aindriu80 Can you move your changes above line 66 please?

Reply:
I've adjusted the location - please check.

Reply:
Awesome, thank you